Add 112's Corona Tournament article (#3611)
ppy/osu-wiki: wiki/Tournaments/112/Corona_2020/en.md

---
tags:
- 112servis
- 112CT
- 112 Corona
---

# 112's Corona Tournament

> ![112CT Logo](img/banner.png)

Reviewer: ```suggestion
![112CT logo](img/banner.png)
```
I think this is supposed to be in sentence case

| Streamer | ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_TR] [HakoFixed](https://osu.ppy.sh/users/7188447) |
| Commentator | ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779), ![][flag_TR] [mezelyus](https://osu.ppy.sh/users/5938859), ![][flag_NL] [goeo_](https://osu.ppy.sh/users/6832107) |
| Designer | ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852) |
| Wiki editor | ![][flag_TR] [Zeus-](https://osu.ppy.sh/users/5464437) |

## Links

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1044086)
- [112's Tournaments Discord server](https://discord.gg/g3nqqeZ)
- [Livestream](https://www.twitch.tv/osu_turkiye)
- [Challonge bracket](https://challonge.com/112korona2020)

## Participants

| Team | Members |
| :-- | :-- |

> | .`**HIDDEN**`.  | ![][flag_TR] [Kizu](https://osu.ppy.sh/users/12940938), ![][flag_TR] [Shinkir0](https://osu.ppy.sh/users/6093148) |

Reviewer: ```suggestion
| .\`\*\*HIDDEN\*\*\`.  | ![][flag_TR] [Kizu](https://osu.ppy.sh/users/12940938), ![][flag_TR] [Shinkir0](https://osu.ppy.sh/users/6093148) |
```
If their team name is ``.`**HIDDEN**`.`` then escape as suggested above

| NM | ![][flag_TR] [Koqla](https://osu.ppy.sh/users/12817802), ![][flag_DE] [Skellexon](https://osu.ppy.sh/users/11268328) |
| Plz Enjoy Chiaki | ![][flag_TR] [Plz Enjoy RSI](https://osu.ppy.sh/users/8365050), ![][flag_TR] [Chiaki Nanami-](https://osu.ppy.sh/users/8588336) |
| SARS-CoV-3 | ![][flag_TR] [Chickes](https://osu.ppy.sh/users/5341891), ![][flag_TR] [AuroraSolaris](https://osu.ppy.sh/users/9319605) | 
| Team Big Chungus  | ![][flag_TR] [pyth0nix](https://osu.ppy.sh/users/10922849), ![][flag_TR] [LeBoum](https://osu.ppy.sh/users/8700026) | 
| Traps Not Gay V2 | ![][flag_TR] [CutieDemon](https://osu.ppy.sh/users/6440158), ![][flag_TR] [HakoFixed](https://osu.ppy.sh/users/7188447) | 
| uwu | ![][flag_TR] [-Miia-](https://osu.ppy.sh/users/8543307), ![][flag_TR] [-Satella-](https://osu.ppy.sh/users/4890227) | 
| Zurna | ![][flag_TR] [Mikatsu](https://osu.ppy.sh/users/6612642), ![][flag_TR] [Coderbora](https://osu.ppy.sh/users/11873333) | 
| ﷽﷽﷽﷽﷽﷽﷽﷽﷽﷽﷽﷽﷽﷽﷽﷽ | ![][flag_TR] [Ekin](https://osu.ppy.sh/users/2947405), ![][flag_TR] [ozge1907](https://osu.ppy.sh/users/6511453) | 

## Podium

> ![112CT Podium](img/podium.png)

Reviewer: ```suggestion
![112CT podium](img/podium.png)
```
ditto

  - [MiYAMO – Kapanet Ntiori (NachoPiggy) \[Overpriced\]](https://osu.ppy.sh/beatmapsets/16606#osu/59420)
  - [Umeboshi Chazuke – Owari to Hajimari no Oto \[Long ver.\] (Azzedd) \[Collab Extra\]](https://osu.ppy.sh/beatmapsets/896054#osu/2001907)
- HardRock
  - [Namaste ft. Tiana Khasi – Signs (Radio Edit) (Xellie) \[pishi's Insane\]](https://osu.ppy.sh/beatmapsets/277479#osu/668108)
  - [Nekomata Master – Sayonara Heaven (Reol) \[Another\]](https://osu.ppy.sh/beatmapsets/535952#osu/1135149)
  - [MASAKI (ZUNTATA) - Scarlet Lance (QuiescentRabbit) \[Expert\]](https://osu.ppy.sh/beatmapsets/967178#osu/2052735)
- DoubleTime
  - [ZUN - Beware the Umbrella Left There Forever (Leyko) \[Nyquill's Lunatic\]](https://osu.ppy.sh/beatmapsets/49852#osu/156070)
  - [fhana - Hoshikuzu no Interlude (Crystal) \[Interlude\]](https://osu.ppy.sh/beatmapsets/225575#osu/526305)
  - [ASCA - Nisemono no Koi ni Sayounara with Wakeshima Kanon (timemon) \[Hard\]](https://osu.ppy.sh/beatmapsets/1055201#osu/2208363)
- FreeMod
  - [Foreground Eclipse – Vermillion Halo (Natteke desu) \[Shiroha my Mirash's Lunatic\]](https://osu.ppy.sh/beatmapsets/1075426#osu/2268407)

>   - [Kaneko Chiharu - iLLness LiLin (Kroytz) \[Pono’s INFINITE\]](https://osu.ppy.sh/beatmapsets/634147#osu/1345674)

Reviewer: ```suggestion
  - [Kaneko Chiharu - iLLness LiLin (Kroytz) \[Pono's INFINITE\]](https://osu.ppy.sh/beatmapsets/634147#osu/1345674)
```
fancy apostrophe to normal

- Hidden
  - [nano.RIPE - Real World (bakabaka) \[Insane\]](https://osu.ppy.sh/beatmapsets/59269#osu/177735)
  - [bibuko - Crayon (val0108) \[Insane\]](https://osu.ppy.sh/beatmapsets/49455#osu/152463)
- HardRock
  - [Rob Gasser – Taking Over (ft. Miyoki) (Mir) \[Conquest\]](https://osu.ppy.sh/beatmapsets/624879#osu/1316977)
  - [GUHROOVY – AIR RAID FROM THA UNDAGROUND (Matrix) \[Niva's Insane\]](https://osu.ppy.sh/beatmapsets/1067854#osu/2314637)
- DoubleTime
  - [wa. – Mellow (Seikatu) \[Another\]](https://osu.ppy.sh/beatmapsets/155094#osu/380596)
  - [Celldweller – The Best It's Gonna Get (Mirash) \[piroshki's Light Insane\]](https://osu.ppy.sh/beatmapsets/1079727#osu/2263669)
  - [cinema staff – great escape (Seto Kousuke) \[Light Insane\]](https://osu.ppy.sh/beatmapsets/915864#osu/1912767)
- FreeMod
  - [COOL&CREATE(BeatMario) – Destined Marionette (avira0922) \[14KEY\]](https://osu.ppy.sh/beatmapsets/8273#osu/40202)
  - [cosMo@Bousou-P - End Mark ni Kibou to Namida wo soete (SnowNiNo\_) \[Internal's Expert\]](https://osu.ppy.sh/beatmapsets/772055#osu/1647542)
- Tiebreaker
  - **[pocotan – Dear Jewel (Meg) \[Jewelry Box\]](https://osu.ppy.sh/beatmapsets/417306#osu/903667)**

> ### Qualifiers stage

Reviewer: ```suggestion
### Qualifiers
```
Other tournaments has it as "Qualifiers", not sure if this is the same or not.

- HardRock
  - [TOTTO – Wadatsumi (Desperate-kun) \[Pho's Insane\]](https://osu.ppy.sh/beatmapsets/351828#osu/775364)
  - [DJ YOSHITAKA – FLOWER (Apricot) \[EXTREME\]](https://osu.ppy.sh/beatmapsets/31054#osu/128780)
- DoubleTime
  - [FELT – Prayer Blue (Frostmourne) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/51145#osu/156927)
  - [LhoU – popotnik \~ The Traveller of Ljubljana (Niva) \[Ryo's Insane\]](https://osu.ppy.sh/beatmapsets/712388#osu/1620017)

## Match results

### Grand Finals

Tuesday, 26 May 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| SARS-CoV-3 | 0 | **7** | **NEŞESİNE YAŞAM** | [#1](https://osu.ppy.sh/community/matches/62136858) |

Saturday, 30 May 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **ali vefa fanclub** | **7** | 1 | NEŞESİNE YAŞAM | [#1](https://osu.ppy.sh/community/matches/62306456) |

### Finals

Monday, 18 May 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **NEŞESİNE YAŞAM** | **6** | 3 | isim bulamadık | [#1](https://osu.ppy.sh/community/matches/61821224) |

Friday, 8 May 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |

> | .`**HIDDEN**`. | 5 | **6** | **isim bulamadık** | [#1](https://osu.ppy.sh/community/matches/61418925) |

Reviewer: ```suggestion
| .\`\*\*HIDDEN\*\*\`. | 5 | **6** | **isim bulamadık** | [#1](https://osu.ppy.sh/community/matches/61418925) |
```

Saturday, 9 May 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| Team Big Chungus | 2 | **6** | **NEŞESİNE YAŞAM** | [#1](https://osu.ppy.sh/community/matches/61461476) |

Sunday, 10 May 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| SARS-CoV-3 | 0 | **6** | **ali vefa fanclub** | [#1](https://osu.ppy.sh/community/matches/61502822) |

### Semifinals

Sunday, 3 May 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| Team Big Chungus | 0 | **1** | **SARS-CoV-3** | *-win by default-* |
| **ali vefa fanclub** | **6** | 3 | isim bulamadık | [#1](https://osu.ppy.sh/community/matches/61222339) |
| **NEŞESİNE YAŞAM** | **6** | 1 | 1.yiz kasmayın | [#1](https://osu.ppy.sh/community/matches/61218558) |

Monday, 4 May 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |

> | **.`**HIDDEN**`.** | **6** | 1 | Zurna | [#1](https://osu.ppy.sh/community/matches/61260110) |

Reviewer: ```suggestion
| **.\`\*\*HIDDEN\*\*\`.** | **6** | 1 | Zurna | [#1](https://osu.ppy.sh/community/matches/61260110) |
```

### Quarterfinals

Saturday, 25 April 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **isim bulamadık** | **5** | 0 | nE | [#1](https://osu.ppy.sh/community/matches/60883287) |

Sunday, 26 April 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **SARS-CoV-3** | **5** | 3 | i may be stupid | [#1](https://osu.ppy.sh/community/matches/60915278) |
| yoo biz 1.yiz | 0 | **1** | **kms** | *-win by default-* |
| **ali vefa fanclub** | **5** | 2 | 1.yiz kasmayın | [#1](https://osu.ppy.sh/community/matches/60921181) |

Saturday, 2 May 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| nE | 3 | **6** | **NEŞESİNE YAŞAM** | [#1](https://osu.ppy.sh/community/matches/61167419) |
| **1.yiz kasmayın** | **6** | 1 | kms | [#1](https://osu.ppy.sh/community/matches/61178173) |

> | i may be stupid | 1 | **6** | **.`**HIDDEN**`.** | [#1](https://osu.ppy.sh/community/matches/61178098) |

Reviewer: ```suggestion
| i may be stupid | 1 | **6** | **.\`\*\*HIDDEN\*\*\`.** | [#1](https://osu.ppy.sh/community/matches/61178098) |
```

| **Zurna** | **6** | 3 | bucocukamerikalı | [#1](https://osu.ppy.sh/community/matches/61181306) |

### Round of 16

Saturday, 18 April 2020:

| Team 1 | | | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |

> | **1.yiz kasmayın** | **5** | 4 | .`**HIDDEN**`. | [#1](https://osu.ppy.sh/community/matches/60565926) |

Reviewer: ```suggestion
| **1.yiz kasmayın** | **5** | 4 | .\`\*\*HIDDEN\*\*\`. | [#1](https://osu.ppy.sh/community/matches/60565926) |
```

Sunday, 19 April 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| yoo biz 1.yiz | 4 | **5** | **SARS-CoV-3** | [#1](https://osu.ppy.sh/community/matches/60613623) |
| **nE** | **5** | 2 | 1deag | [#1](https://osu.ppy.sh/community/matches/60618664) |
| kms | 4 | **5** | **i may be stupid** | [#1](https://osu.ppy.sh/community/matches/60624640) |
| **Team Big Chungus** | **5** | 1 | Mağdurlar | [#1](https://osu.ppy.sh/community/matches/60624863) |

Monday, 20 April 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **isim bulamadık** | **5** | 1 | bucocukamerikalı | [#1](https://osu.ppy.sh/community/matches/60665808) |

Saturday, 25 April 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **bucocukamerikalı** | **5** | 2 | 1deag | [#1](https://osu.ppy.sh/community/matches/60886113) |

Sunday, 26 April 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| Mağdurlar | 3 | **5** | **NEŞESİNE YAŞAM** | [#1](https://osu.ppy.sh/community/matches/60919257) |

Monday, 27 April 2020:

| Team 1 |  |  | Team 2 | Match link |
| --: | :-: | :-: | :-- | :-- |

> | Traps Not Gay V2 | 1 | **5** | **.`**HIDDEN**`.** | [#1](https://osu.ppy.sh/community/matches/60966668) |

Reviewer: ```suggestion
| Traps Not Gay V2 | 1 | **5** | **.\`\*\*HIDDEN\*\*\`.** | [#1](https://osu.ppy.sh/community/matches/60966668) |
```

## Ruleset

### Tournament rules

> 1. 112’s Corona Tournament is an osu!standard tournament only allowed for Turkish-speaking players regardless of their profile flags.

Reviewer: ```suggestion
1. 112's Corona Tournament is an osu!standard tournament only allowed for Turkish-speaking players regardless of their profile flags.
```
ditto

14. Players that are suspected of cheating during the tournament will be disqualified from the tournament.
15. Tournament hosts hold the right to exclude any player from participating in the tournament at will.

### Registration

1. Every user interested in joining the tournament will sign up individually.
   - **Any form of registration, other than using the method provided in related channels, will be disregarded.**
2. Registrations will be handled in the [Discord server](https://discord.gg/g3nqqeZ) via a special tournament bot. Instructions are provided in related channels. Participants **must not** leave the server for the duration of the tournament.
3. As no badge will be awarded, tournament staff can participate in the tournament. Screening procedure will also not be applied for this tournament.
4. Participants will be published after the Registration period ends.

### Stage Instructions

1. In the Qualifier stage, each seed will have a specific pool designed by the map selectors.

> 2. The pool will contain 4 brackets: NoMod, [Hidden](wiki/Game_Modifiers), [HardRock](wiki/Game_Modifiers), [DoubleTime](wiki/Game_Modifiers). There will not be a tiebreaker for this stage.

Reviewer: ```suggestion
2. The pool will contain 4 brackets: NoMod, [Hidden](/wiki/Game_Modifiers), [HardRock](/wiki/Game_Modifiers), [DoubleTime](/wiki/Game_Modifiers). There will not be a tiebreaker for this stage.
```

6. Players will have the option to choose the multiplayer room they wish to play. If a room happens to be too empty, remaining players will be navigated to the other rooms.
7. Latecomers can be admitted to the room, but they have to commence playing from the part where the others are at.
8. If no players will join the room, the match will be aborted. No points will be awarded to the absent players.
9. First 16 players with the highest overall score will get through to Double Elimination stages. If there will be more participants than the predicted number, player(s) placed the last will be eliminated. 
10. During the Double Elimination stages, the winner moves to the next stage and the losing team gets moved to the Losers bracket.
11. **Winning conditions:**
    - The Round of 16 and Quarterfinals will be best of 9 (first team to 5 wins).        
    - The Semifinals and Finals will be best of 11 (first team to 6 wins).        
    - The Grand Finals will be best of 13 (first team to 7 wins).

### Match instructions

1. A referee will create a multiplayer room 15 minutes in advance. Players must gather during this period.

>    - Room settings are osu!, Team-Vs. , Win Condition: ‘ScoreV2’.

Reviewer: ```suggestion
   - Room settings are osu!, Team-Vs., Win Condition: 'ScoreV2'.
```

3. Each team must use `!roll` once in `#multiplayer`.
   - The team with the higher roll **picks** first.       
   - The team with the lower roll **bans** first.
   - Each team will be given 2 minutes to decide upon picks or bans. If first team fails to provide an answer, other team takes the turn.
   - **From Quarterfinals onwards the team that bans first will ban one map, then the other team will ban two maps, then finally the first team will make their final ban.**
4. During Round of 16, each player can only ban 1 beatmap.
   - **From Quarterfinals onwards, the number of bans increases to 2. However, each player/team may not ban more than one map of any mod bracket (excluding NoMod).**
5. Beatmap selection will alternate between each player/team selecting a beatmap out of the mappool.
6. Players map pick freely from any mod pool.
   - In the case of a tie, the tiebreaker beatmap will be played.
7. The results will be published in the dedicated Discord channel in the server.

### Mappool Instructions

1. There will be a new mappool for every stage.
   - Only Finals and Grand Finals will share the same pool.

> 2. Each mappool consists of 5 brackets: NoMod, [Hidden](wiki/Game_Modifiers), [HardRock](wiki/Game_Modifiers), [DoubleTime](wiki/Game_Modifiers) and FreeMod.

Reviewer: ```suggestion
2. Each mappool consists of 5 brackets: NoMod, [Hidden](/wiki/Game_Modifiers), [HardRock](/wiki/Game_Modifiers), [DoubleTime](/wiki/Game_Modifiers) and FreeMod.
```
